R_program/assignsiteid.py

```diff
@@ -34,6 +34,15 @@
 f.write(newdata)
 f.close()
 
+fileout='STEPWAT_DIST/sample.sh'
```

Comment:
Because I made assignsiteid.py object-oriented in the streamline_output branch all of these changes can be compressed into `myParser.Parse('STEPWAT_DIST/sample.sh')` once these branches are merged.

Reply:
@chaukap I am not entirely sure I understand what you are recommending here...

Reply:
Sorry for the vagueness! In the [streamline_output branch](https://github.com/DrylandEcology/rSFSTEP2/blob/streamline_output/R_program/assignsiteid.py) I removed all of the redundancies in this script by reading a parsing object called `myParser`. `myParser.Parse(fileName)` is a function that performs all of the replacements like "notassigned" and "noid". 

_All_ changes on this branch to assignsiteid.py can be replaced by this one line.